proto/osmosis/poolmanager/v1beta1/tx.proto

```diff
@@ -114,3 +116,26 @@ message MsgSplitRouteSwapExactAmountOutResponse {
     (gogoproto.nullable) = false
   ];
 }
+
+// ===================== MsgSetDenomPairTakerFee
+message MsgSetDenomPairTakerFee {
+  option (amino.name) = "osmosis/poolmanager/set-denom-pair-taker-fee";
+
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  repeated DenomPairTakerFee denom_pair_taker_fee = 2 [
+    (gogoproto.moretags) = "yaml:\"denom_pair_taker_fee\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message MsgSetDenomPairTakerFeeResponse { bool success = 1; }
+
+message DenomPairTakerFee {
```

Comment:
Lets add a comment here if there is restriction's on the ordering of denom0 / denom1.

Reply:
No restriction, will add comment showing this

Reply:
Added https://github.com/osmosis-labs/osmosis/pull/6034/commits/73830bcd7d6bae9e8dcbe4d41d03ee658b9bc55c